generator-theia/src/extension/extension-package-generator.ts

```diff
@@ -57,6 +57,15 @@ export class ExtensionPackageGenerator extends AbstractGenerator {
     protected compileCommonPackage(): NodePackage {
         const keywords = new Set(this.model.pck.keywords || []);
         keywords.add(this.model.extensionConfig.extensionKeyword);
+        let buildCmd = "";
+        if (this.model.extensionConfig.localize) {
+            const split = `${process.cwd()}`.split("/");
+            const processName = split.reverse()[0];
+            buildCmd = "localize-theia " + `${processName}` + " && concurrently -n compile,lint -c blue,green \"yarn run compile\" \"yarn run lint\"";
```

Comment:
This is not the process name but the package name right ?

You should use node's path class see: https://nodejs.org/dist/latest-v8.x/docs/api/path.html to make this cross platform

Reply:
Also the concurrently compile etc.. line is repeated... it should be in it's own var

Reply:
Done, use path to extract the package name, use a const for the build command.